seed/static/seed/js/controllers/export_buildingsync_modal_controller.js

```diff
@@ -10,22 +10,29 @@ angular.module('BE.seed.controller.export_buildingsync_modal', [])
     '$uibModalInstance',
     'property_view_id',
     'column_mapping_profiles',
+    'user_service',
     function (
       $http,
       $window,
       $scope,
       $uibModalInstance,
       property_view_id,
-      column_mapping_profiles
+      column_mapping_profiles,
+      user_service,
     ) {
       $scope.column_mapping_profiles = column_mapping_profiles;
       $scope.current_column_mapping_profile = column_mapping_profiles[0];
 
       $scope.download_file = function () {
```

Comment:
Please test this by going to inventory detail page, then selecting export as buildingsync action

Reply:
Works as expected 👍